services/web-server/src/resolvers/Hooks.js

```diff
@@ -21,6 +21,7 @@ export default {
     SCHEDULE: 'schedule',
     TRIGGER_HOOK: 'triggerHook',
     TRIGGER_HOOK_WITH_TOKEN: 'triggerHookWithToken',
+    PULSE_MESSAGE: 'pulseMessage',
```

Comment:
Where is this defined in the schema? A link to the schema file would be helpful.

Reply:
Actually `firedBy` defined as any string in the schema (should change to enum types..) https://docs.taskcluster.net/docs/reference/core/hooks/api#listLastFires
Check here: https://docs.taskcluster.net/docs/reference/core/hooks/firing-hooks#pulse-messages

Reply:
:100: to changing those to enum types